conftest.py

```diff
@@ -2,6 +2,8 @@
 
 import pytest
 
+import borg.cache
```

Comment:
The reason I originally put it into the plugin function was that this way it's impossible to hit by accident (say, importing something from the conftest in test code, and maybe that is hit during self-test and maybe not immediately noticed, because py.test is installed). Call it ~~defensive~~ paranoid programming ;)

The resolution is ok, perhaps I'll send a patch later that restores the paranoia.